src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -125,23 +125,18 @@ void Terminal::UpdateSettings(ICoreSettings settings)
 
     if (settings.TabColor() == nullptr)
     {
-        _tabColor = std::nullopt;
+        _renderSettings.SetColorTableEntry(TextColor::FRAME_BACKGROUND, INVALID_COLOR);
     }
     else
     {
-        _tabColor = settings.TabColor().Value();
+        _renderSettings.SetColorTableEntry(TextColor::FRAME_BACKGROUND, til::color{ settings.TabColor().Value() });

```

Comment:
I don't 100% understand the color table vs alias thing right now, but should this use `SetColorAlias` to both (1) point `FrameBackground` at `FRAME_BACKGROUND` _and_ (2) set `FRAME_BACKGROUND` to the new tab color?

Or is the intent that once redirected via `DECAC`, user-specified tab color cannot override application-specified tab color?

Reply:
(And keeping with my (potential mis-)understanding of color aliases:) If that _is_ the intent, then updating only the `FRAME_BACKGROUND` color but not re-pointing `FrameBackground` back to it is in line with that intent (?)

Reply:
I'm glad you raised this, because I had meant to bring it up myself. I'm open to changing this, but I did it this way because it lets you map the frame background alias to the default background color table (using `DECAC 2;261;262`), and then switch your color scheme in the settings and still have both remain in sync. If we reset the frame alias here then we lose that ability.

But further down the line I'm assuming you might add an option for users to explicitly request that behavior (to support #702), and then the decision as to what should happen here would probably be controlled by that setting.

Does that make sense?

Reply:
> Does that make sense?

100%. We can refine this if/when we add this part of 702. For now, I like how it is. Thanks!

Mike may be changing the default s.t. Terminal always acts like `DECAC 2;261;262` over in #12992, or shortly after. /cc @zadjii-msft for this discussion.